src/com/estafet/learning/US008/Order.java

```diff
@@ -0,0 +1,27 @@
+package com.estafet.learning.US008;
+
+public abstract class Order {
+    public static final String ORDER_NAME = "ORR";
+    public String OrderNumber;
+    public String ClientDetails;
+    public String ListWithArticles;
+    public String TotalAmount;
+    private static final int invoice = 1;
+    public static final String color = "green";
+
+    public Order(){
+    }
+
+    public Order(String orderNumber, String clientDetails){
+        this.OrderNumber = orderNumber;
+        this.ClientDetails = clientDetails;
+    }
+
+    public Order(String orderNumber,String clientDetails,String listWithArticles,String totalAmount){
```

Comment:
set up automatic code reformat in IDE so it will fix missing spaces, identations etc. before commits automatically

Reply:
changed.